README.md

```diff
@@ -63,6 +63,8 @@ What are embeddings?
 - __Technical__: An embedding is the latent-space position of a document at a layer of a deep neural network. For models trained specifically to embed data, this is the last layer.
 - __A small example__: If you search your photos for "famous bridge in San Francisco". By embedding this query and comparing it to the embeddings of your photos and their metadata - it should return photos of the Golden Gate Bridge.
 
+## Deployment
```

Comment:
@ericmand would you be amenable to remove this from the main README? We can find the right place in the docs. Since we have a large refactor coming, https://github.com/chroma-core/chroma/pull/214, I want to let users know this and help them understand our commitment to making the transition seamless. But the readme isn't the right place to dump all that context. wdyt?

Reply:
Done!

Reply:
thanks! will get this merged once tests pass and then get it reflected over the docs https://github.com/chroma-core/docs/pull/16